src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -190,8 +190,10 @@ protected override void Apply(TransactionOperationContext context, BlittableJson
                     case nameof(UpdateRavenEtlCommand):
                     case nameof(UpdateSqlEtlCommand):
                     case nameof(DeleteOngoingTaskCommand):
-                    case nameof(PutRavenConnectionString):
-                    case nameof(PutSqlConnectionString):
+                    case nameof(PutRavenConnectionStringCommand):
+                    case nameof(PutSqlConnectionStringCommand):
+                    case "PutRavenConnectionString":   // Legacy
+                    case "PutSqlConnectionString":     // Legacy
                     case nameof(RemoveRavenConnectionString):
                     case nameof(RemoveSqlConnectionString):
```

Comment:
So these two should be renamed as well (`RemoveRavenConnectionString, RemoveSqlConnectionString`),

Reply:
They are, I just adding support for the old name, in case users would upgrade just one server in the database and this command would made it in to the log of the server that wasn't updated.

Reply:
I meant, shouldn't we also do the following renames?:
- `RemoveRavenConnectionString -> RemoveRavenConnectionStringCommand`
- `RemoveSqlConnectionString -> RemoveSqlConnectionStringCommand`

Reply:
Oh, I'll rename them as well. I didn't touched them so I didn't see them. Good catch.